### ISSUE TITLE
Cant find_package(PugiXML REQUIRED) twice

### ISSUE THREAD

**Issue by LocutusOfBorg** [zeux/pugixml#393]
This happens since 1.11.1 and is a regression because it makes some external packages fails the autopkgtests
e.g.
https://ci.debian.net/data/autopkgtest/testing/amd64/i/ismrmrd/9008335/log.gz
Because of code like this:
```
if(USE_SYSTEM_PUGIXML)
   find_package(PugiXML CONFIG)
   if (NOT PUGIXML_FOUND)
     find_dependency(PugiXML)
   endif()
endif()
```

cat CMakeLists.txt
```
project(test)
cmake_minimum_required(VERSION 3.18)

find_package(PugiXML REQUIRED)
find_package(PugiXML REQUIRED)
find_package(PugiXML REQUIRED)
find_package(PugiXML REQUIRED)
find_package(PugiXML REQUIRED)
```

```
cmake ..
-- The C compiler identification is GNU 10.2.1
-- The CXX compiler identification is GNU 10.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/pugixml/pugixml-config.cmake:26 (add_library):
  add_library cannot create imported target "pugixml" because another target
  with the same name already exists.
Call Stack (most recent call first):
  CMakeLists.txt:5 (find_package)


CMake Error at /usr/lib/x86_64-linux-gnu/cmake/pugixml/pugixml-config.cmake:26 (add_library):
  add_library cannot create imported target "pugixml" because another target
  with the same name already exists.
Call Stack (most recent call first):
  CMakeLists.txt:6 (find_package)


CMake Error at /usr/lib/x86_64-linux-gnu/cmake/pugixml/pugixml-config.cmake:26 (add_library):
  add_library cannot create imported target "pugixml" because another target
  with the same name already exists.
Call Stack (most recent call first):
  CMakeLists.txt:7 (find_package)


CMake Error at /usr/lib/x86_64-linux-gnu/cmake/pugixml/pugixml-config.cmake:26 (add_library):
  add_library cannot create imported target "pugixml" because another target
  with the same name already exists.
Call Stack (most recent call first):
  CMakeLists.txt:8 (find_package)


-- Configuring incomplete, errors occurred!
See also "/ismrmrd-1.4.2.1/foo/bar/foo/CMakeFiles/CMakeOutput.log".
```

**Comment by LocutusOfBorg** [comment]
there is also a problem in ismrmrd because PugiXML_FOUND is not the same as PUGIXML_FOUND

**Comment by zeux** [comment]
Thanks, confirmed - doing find_package twice in a simple example produces the same CMake error as above. PR incoming, and I will tag yet another patch release after it gets merged.

**Comment by zeux** [comment]
This issue has been fixed now and v1.11.4 has been tagged.
